crates/chain/src/tx_graph.rs

```diff
@@ -1066,6 +1066,33 @@ impl<A> ChangeSet<A> {
             })
             .chain(self.txouts.iter().map(|(op, txout)| (*op, txout)))
     }
+
+    /// Iterates over the heights of that the new transaction anchors in this changeset.
+    ///
+    /// This is useful if you want to find which heights you need to fetch data about in order to
+    /// confirm or exclude these anchors.
+    ///
+    /// See also: [`Self::missing_heights`]
+    pub fn relevant_heights(&self) -> impl Iterator<Item=u32> + '_ where A: Anchor {
```

Comment:
**Nit:** I would name this `anchor_heights`. I think `relevant` here seems ambiguous.